backend/scrapers/police_coders_groups/scraper.py

```diff
@@ -1,42 +1,40 @@
-from __future__ import print_function
 import pickle
 import os.path
 import csv
 from googleapiclient.discovery import build
 from google_auth_oauthlib.flow import InstalledAppFlow
 from google.auth.transport.requests import Request
+from datetime import datetime
 
-
-def googleScrape(URL, SpreadsheetID, SpreadsheetRange, path_creds, path_out):
+def googleScrape(filename):
 
     # If modifying these scopes, delete the file token.pickle.
-    SCOPES = [URL]
+    SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly']
 
     # The ID and range of a sample spreadsheet.
-    SAMPLE_SPREADSHEET_ID = SpreadsheetID
-    SAMPLE_RANGE_NAME = SpreadsheetRange
+    SAMPLE_SPREADSHEET_ID = '1iqOvNjRlHIpoRzd61BcBLVkSxGvbta6vrzH2Jgc50aY'
+    SAMPLE_RANGE_NAME = 'Support groups v2'
 
     def main():
```

Comment:
I see a main here so I would've expected to find this as main script - then located in `/backend/` - but it does not seem the case. 

Then, this either is a module or a _standalone_ main script expected to be executed in isolation? 
pick one and modify accordingly either renaming or moving the file ;)